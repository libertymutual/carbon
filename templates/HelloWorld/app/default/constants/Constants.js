// Platform Types
export const OS_IOS = 'iOS';
export const OS_ANDROID = 'Android';

// EXTERNAL FEATURE & MODULE NAMES
export const EXTERNAL_FEATURE_SECOND = 'Second';
export const EXTERNAL_FEATURE_MODULE_SECOND = 'SecondModule';

export const EXTERNAL_FEATURE_WEB = 'Web';
export const EXTERNAL_FEATURE_MODULE_WEB = 'WebModule';
