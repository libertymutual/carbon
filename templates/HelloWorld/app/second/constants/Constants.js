// Platform Types
export const OS_IOS = 'iOS';
export const OS_ANDROID = 'Android';

// HELLO WORLD STATES
export const HELLO = 'hello';
export const GOODBYE = 'goodbye';

// EXTERNAL FEATURE & MODULE NAMES
export const EXTERNAL_FEATURE_DEFAULT = 'Default';
export const EXTERNAL_FEATURE_MODULE_DEFAULT = 'DefaultModule';
