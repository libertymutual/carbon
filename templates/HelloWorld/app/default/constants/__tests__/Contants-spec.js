import 'react-native';
import React from 'react';
import * as Constants from '../Constants';

describe('Constants tests', () => {

  test('OS_IOS returns correct value', () => {
    expect(Constants.OS_IOS).toMatchSnapshot();
  });

  test('OS_ANDROID returns correct value', () => {
    expect(Constants.OS_ANDROID).toMatchSnapshot();
  });

  test('EXTERNAL_FEATURE_SECOND returns correct value', () => {
    expect(Constants.EXTERNAL_FEATURE_SECOND).toMatchSnapshot();
  });

  test('EXTERNAL_FEATURE_MODULE_SECOND returns correct value', () => {
    expect(Constants.EXTERNAL_FEATURE_MODULE_SECOND).toMatchSnapshot();
  });

  test('EXTERNAL_FEATURE_WEB returns correct value', () => {
    expect(Constants.EXTERNAL_FEATURE_WEB).toMatchSnapshot();
  });

  test('EXTERNAL_FEATURE_MODULE_WEB returns correct value', () => {
    expect(Constants.EXTERNAL_FEATURE_MODULE_WEB).toMatchSnapshot();
  });
});
