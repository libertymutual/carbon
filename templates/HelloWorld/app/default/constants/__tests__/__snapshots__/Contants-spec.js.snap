// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants tests EXTERNAL_FEATURE_MODULE_SECOND returns correct value 1`] = `"SecondModule"`;

exports[`Constants tests EXTERNAL_FEATURE_MODULE_WEB returns correct value 1`] = `"WebModule"`;

exports[`Constants tests EXTERNAL_FEATURE_SECOND returns correct value 1`] = `"Second"`;

exports[`Constants tests EXTERNAL_FEATURE_WEB returns correct value 1`] = `"Web"`;

exports[`Constants tests OS_ANDROID returns correct value 1`] = `"Android"`;

exports[`Constants tests OS_IOS returns correct value 1`] = `"iOS"`;
